### DISCUSSION TITLE
OKD 4.14 vSphere UPI Bootstrap Fail

### DISCUSSION THREAD

**Discussion by Hyun-June-Choi** [okd-project/okd#1829] [Q&A]
openshift-install-linux-4.14.0-0.okd-2023-12-01-225814

install with openshift-install(no agent)
OKD 4.14 vSphere UPI Bootstrap Fail,  log bundle attached

FullName:     VMware vCenter Server 8.0.2 build-22617221
Name:         VMware vCenter Server
Vendor:       VMware, Inc.
Version:      8.0.2
Build:        22617221
OS type:      linux-x64
API type:     VirtualCenter
API version:  8.0.2.0

[install-config.yaml]

additionalTrustBundlePolicy: Proxyonly
apiVersion: v1
baseDomain: best.id
proxy:
  httpProxy: http://10.25.88.120:3128
  httpsProxy: http://10.25.88.120:3128
  noProxy: localhost,.best.id,.svc,vcdc01.hanabank.co.id,4.4.4.14,10.25.90.0/24,10.87.190.0/24,10.133.161.0/24,10.25.151.0/24,10.25.152.10 
compute:
- architecture: amd64
  hyperthreading: Enabled
  name: worker
  platform: {}
  replicas: 0
controlPlane:
  architecture: amd64
  hyperthreading: Enabled
  name: master
  platform: {}
  replicas: 3
metadata:
  creationTimestamp: null
  name: prod
networking:
  clusterNetwork:
  - cidr: 10.128.0.0/14
    hostPrefix: 23
  machineNetwork:
  - cidr: 10.0.0.0/16
  networkType: OVNKubernetes
  serviceNetwork:
  - 172.30.0.0/16
platform:
  vsphere:
    apiVIPs:
    - 10.133.161.35
    failureDomains:
    - name: generated-failure-domain
      region: generated-region
      server: vcdc01.hanabank.co.id
      topology:
        computeCluster: /DC-NTT/host/BEST-CLUSTER
        datacenter: DC-NTT
        datastore: /DC-NTT/datastore/BEST-DATASTORE/BEST-DATASTORE01
        networks:
        - vlan61
        resourcePool: /DC-NTT/host/BEST-CLUSTER//Resources
      zone: generated-zone
    ingressVIPs:
    - 10.133.161.34
    vcenters:
    - datacenters:
      - DC-NTT
      password: *********
      port: 443
      server: *******************
      user: *****************
publish: External
pullSecret: '{"auths":{"fake":{"auth":"aWQ6cGFzcwo="},"mirror.prod.best.id:8443":{"auth":"aW5pdDozMkpIUTY4TVIwN3ZjeXRsWklhNGd4emtCMU5QaTVMOQ=="}}}'
sshKey: |
  ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAICJ3kK46ZLo+x7Ye7JYlbpFX3d/CXbhs66oEW7InWg3C itdev@bastion.prod.best.id
additionalTrustBundle: | 
  -----BEGIN CERTIFICATE-----
  MIID3zCCAsegAwIBAgIUUXiscN05ASf6bZX2FPGTHf1ZTtgwDQYJKoZIhvcNAQEL
  BQAwbTELMAkGA1UEBhMCVVMxCzAJBgNVBAgMAlZBMREwDwYDVQQHDAhOZXcgWW9y
  azENMAsGA1UECgwEUXVheTERMA8GA1UECwwIRGl2aXNpb24xHDAaBgNVBAMME21p
  cnJvci5wcm9kLmJlc3QuaWQwHhcNMjMxMjA5MDMzMTQ4WhcNMjYwOTI4MDMzMTQ4
  WjBtMQswCQYDVQQGEwJVUzELMAkGA1UECAwCVkExETAPBgNVBAcMCE5ldyBZb3Jr
  MQ0wCwYDVQQKDARRdWF5MREwDwYDVQQLDAhEaXZpc2lvbjEcMBoGA1UEAwwTbWly
  cm9yLnByb2QuYmVzdC5pZDCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEB
  AIXId2DXvA306eQ+ZLPx87Za8RAtQQOoKOhNMPzpwJbUwGHiYrSFUQdlBo8g2nVk
  lcZDRHkF0v2ZjH81l6/th1z8vN934rCAgx6QPmuM4MCzCFmH3xHf7vPN32XHNX9T
  BhcFA1/i+K1zi80oLvXF80Ikcs5IwXISZfuFeQbZfsqCo2durorYrI2Yl340Pv24
  GZ7U9aXMYVMOdH/kuVUnS11yxpfJ873PLYYBVtAdTl9b7fZuiWn4FC45r/hGFLm9
  jfpx/ELY3lksbeqEqbcHKJH75/4fqymCFS/qFWlIn9LxKFNZcmt9W2vD+zCKZdU2
  CQwACrmdGKecHXLqRHsklwUCAwEAAaN3MHUwCwYDVR0PBAQDAgLkMBMGA1UdJQQM
  MAoGCCsGAQUFBwMBMB4GA1UdEQQXMBWCE21pcnJvci5wcm9kLmJlc3QuaWQwEgYD
  VR0TAQH/BAgwBgEB/wIBATAdBgNVHQ4EFgQUpLF8Cjc/NfyZEwzJ3cLPlB0+BFkw
  DQYJKoZIhvcNAQELBQADggEBAE0aO9pKk8Ff8QYUTOgdBY5gg/4hBAaJwoZsUEzK
  QDefeGfGuEYhuyWf+hMVRfGIQhwHdbAW74toRATBr4FYu16wtSGMSLB8R6M3pHvQ
  gsrCWmlVqzFYnhCNfAt2KioML5HT9X4wBmjOKWvF9eN8fC28l8c/zqR7p5CHL3X6
  jojjWE/NgdlPEc4w+JBJ3Y9jiq5XtGB+jQcpQ3/B2gbLJKflPF21Q1vdRrZSbLCt
  qNvvTHpY8ENF+HAvg4fHDSgI470k07ZhBlBHD97rZXWJK6H9fxCbv0NqnaU5IYkK
  skpvLTwFOFUXAhpDAYGwewSjHA/v9xBBfb0tqu20zz30CZk=
  -----END CERTIFICATE-----
imageDigestSources: 
- mirrors:
  - mirror.prod.best.id:8443/origin/okd
  source: quay.io/openshift/okd
- mirrors:
  - mirror.prod.best.id:8443/origin/okd
  source: quay.io/openshift/okd-content

[log-bundle-20231213115202.tar.gz](https://github.com/okd-project/okd/files/13656308/log-bundle-20231213115202.tar.gz)

**Comment by jcpowermac** [comment]
The control plane nodes are not igniting

**Reply by Hyun-June-Choi** [reply]
could you explain details?

**Reply by jcpowermac** [reply]
Check the console of your control plane virtual machines. I would guess they are sitting there looping to download ignition.

They never contacted the machine config server so they didn't perform the even basic configuration.
